docs/user_guide/Upgrade_SIMP/Version_Maps/6.3.3_6.4.0.inc

```diff
@@ -30,25 +30,33 @@ used to.
 To get a full accounting of the major changes in behavior for 6.4.0, please see
 the :ref:`changelog-6.4.0` Changelog.
 
-Package Upgrade Order
-^^^^^^^^^^^^^^^^^^^^^
+.. IMPORTANT::
+
+  This upgrade has a great potential for deleting files you need.  Read the
+  instructions carefully and back up your system.
+
+RPM Name Changes and the Consequences
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 simp-rsync
 """"""""""
 
+simp-rsync has been renamed to simp-rsync-skeleton.
```

Comment:
Since we have one last simp-rsync package, does it make sense to say the package has been renamed or should we say it has been replace by simp-rsync-skeleton, moving forward?